pkgs/applications/science/electronics/pulseview/default.nix

```
@@ -1,45 +1,38 @@
-{ lib, stdenv, fetchurl, fetchpatch, pkg-config, cmake, glib, boost, libsigrok
-, libsigrokdecode, libserialport, libzip, udev, libusb1, libftdi1, glibmm
-, pcre, python3, qtsvg, qttools, wrapQtAppsHook, desktopToDarwinBundle
+{ lib, stdenv, fetchgit, pkg-config, cmake, glib, boost, libsigrok
+, libsigrokdecode, libserialport, libzip, libftdi1, hidapi, glibmm
+, pcre, python3, qtsvg, qttools, bluez, util-linux, libselinux, libsepol
+, wrapQtAppsHook, desktopToDarwinBundle
 }:
 
 stdenv.mkDerivation rec {
   pname = "pulseview";
-  version = "0.4.2";
+  version = "nightly-2023-06-16";
```

Looking through nixpkgs, `"unstable-YYYY-MM-DD"` is much more common than `"nighly-YYYY-MM-DD"`.

```suggestion
  version = "unstable-2023-06-16";
```

(also for the other files and commit msgs)

---

Oh, yes, that is true.

However, the project themselves calls it nightly and there was a lot of heated discussion about the nightlies being called "unstable", I think some people might feel attacked.
This is what a project member wanted it to be called.

So, IMO, better to stick with nightly.